docs/source/_templates/docs-sidebar.html

```diff
@@ -1,6 +1,6 @@
 
 <a class="navbar-brand" href="{{ pathto(master_doc) }}">
-  <img src="{{ pathto('_static/' + logo_url, 1) }}" class="logo" alt="logo">
+  <img src="{{ logo_url|e }}" class="logo" alt="logo">
```

Comment:
Does this work only with Sphinx 5?
If this works with both of Sphinx 5 and 6, how about removing `,<6` for `sphinx` from this PR?

Reply:
This works for both but there are other issue (#35403) that require us to pin to <6 for now.

Reply:
Yes, @assignUser is spot on. This fixes the logo issue (https://github.com/apache/arrow/issues/35406) regardless of the sphinx version, but we still need to fix the jquery dependency issue for being able to use sphinx 6

Reply:
Thanks. I see.